enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/utils/default-embed-setting.ts

```diff
@@ -45,6 +46,15 @@ export const getDefaultSdkIframeEmbedSettings = (
         isSaveEnabled: false,
       }),
     )
+    .with(
+      "browser",
+      (): BrowserEmbedOptions => ({
+        componentName: "metabase-browser",
+        initialCollection: "root",
+        readOnly: true,
+        collectionVisibleColumns: ["type", "name"],
```

Comment:
We should hide `Last edited by` and `Last edited at` by default in embedding environments.

Reply:
I recalled Alberto wants to have this as a default for the browser templates, but at the same time if we modify this in the CollectionBrowser default it will a breaking change, and if we change it in custom element default it will get out of sync with the SDK. I think simply saying something in the UI might be the way to go here.

Reply:
That makes sense! I will make the browser component itself default to `collectionVisibleColumns: ["type", "name"]` then. I'm thinking if we should still show it in the snippet for clarity though, so people know that's the default.